repos/c2sm/packages/py-icon4py/package.py

```diff
@@ -22,6 +22,8 @@ class PyIcon4py(PythonPackage):
     version('main', branch='main', git=git)
     version('0.0.3', tag='v0.0.3', git=git)
     version('0.0.4', tag='v0.0.4', git=git)
+    version('0.0.5', tag='v0.0.5', git=git)
+    version('0.0.6', tag='v0.0.6', git=git)
```

Comment:
```suggestion
```
I recommend to add this version only if you plan to tag icon4py after your PR.
Otherwise this is not needed.

Reply:
Yes I will tag it